### PR TITLE
Added basic support for Omnitron iConverters

### PR DESCRIPTION
**Pull request by EnzoZafra**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

fixes: #6329

### CODE REVIEW

File: includes/definitions/omnitron-iconverter.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+icon: omnitron
+mib_dir:
+    - omnitron
+#over:
+    #- { graph: device_current, text: 'Current' }
+    #- { graph: device_voltage, text: 'Voltage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7342
+    - sysDescr_regex:
```
Comment (murrant) [inline includes/definitions/omnitron-iconverter.yaml]: Please remove sysDescr_regex.
What you have here is:
sysObjectId starts with .1.3.6.1.4.1.7342
OR
sysDescr matches this regex "..."

If you want
sysObjectId starts with .1.3.6.1.4.1.7342
AND
sysDescr contains iConverter
Use this:
```
discovery:
    -
        - sysObjectId: .1.3.6.1.4.1.7342
        - sysDescr: iConverter
```

But it looks like the vendor has properly utilized sysObjectId, so you can just use:
```
discovery:
    - sysObjectId:
        - .1.3.6.1.4.1.7342.5.
```

**Comment by murrant** [comment]
Can you please crop the logo png?

And fix the discovery issue I mentioned.  After that, this is good to merge.

**Comment by EnzoZafra** [comment]
Done. Sorry, new to this.

**Comment by murrant** [comment]
No worries, looks good overall, just a few items.
Documentation on the yaml discovery isn't the clearest.

**Pull request merged**